.mci.yml

```diff
@@ -877,6 +877,7 @@ buildvariants:
             - ubuntu1804-build
           - name: uninstall_check
 
+    # TODO CXX-2260 Upgrade std::experimental tasks to Ubuntu 18.04 after bug fixes with g++ 5.5
```

Comment:
Should this variant also be changed to test against 4.4 instead of latest in the meantime?

```
mongodb_version: *version_44
```

IIRC otherwise versioned API tests against the latest server version published on Ubuntu 16.04 may not pass.

Reply:
Ah great point. Used `version_44` and updated comment and `display_name`.